e2e/ci-e2e/tests/ci.e2e.test.ts

```diff
@@ -0,0 +1,180 @@
+import {
+  copyFile,
+  mkdir,
+  readFile,
+  rename,
+  rm,
+  writeFile,
+} from 'node:fs/promises';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import {
+  type DiffResult,
+  type FetchResult,
+  type SimpleGit,
+  simpleGit,
+} from 'simple-git';
+import {
+  type Comment,
+  type GitRefs,
+  type Options,
+  type ProviderAPIClient,
+  type RunResult,
+  runInCI,
+} from '@code-pushup/ci';
+
+describe('CI package', () => {
+  const fixturesDir = join(
+    fileURLToPath(dirname(import.meta.url)),
+    '..',
+    'mocks',
+    'fixtures',
+  );
+  const workDir = join(process.cwd(), 'tmp', 'ci-e2e', 'test-repo');
```

Comment:
For test folder we follow the pattern: `tmp/e2e/<project-identifier>/<test-case>`

I suggest:
1. `tmp/e2e/ci-e2e/<test-case>` for a setup per it block
2. `tmp/e2e/ci-e2e/test-repo/<test-case>` for a setup where it blocks depend on `test-repo` but not modify it
3. `tmp/e2e/ci-e2e/<test-case>-test-repo` where the test repo folder gets modified in any way

I suggest 3.